src/theme/dojo/popup-confirmation.m.css

```diff
@@ -0,0 +1,68 @@
+:root {
```

Comment:
Please do not create any `:root` level variables, just hardcode this value where it's used if it cannot be effected by a theme